src/placement/jump_map.c

```diff
@@ -898,27 +903,25 @@ jump_map_obj_place(struct pl_map *map, struct daos_obj_md *md,
 		D_INIT_LIST_HEAD(&add_list);
 		layout_find_diff(jmap, layout, add_layout, &add_list);
 
-		if (!d_list_empty(&add_list)) {
-
+		if (!d_list_empty(&add_list))
 			rc = pl_map_extend(layout, &add_list, true);
-			if (rc != 0)
-				return rc;
```

Comment:
might change it to "goto out", just in case we add more code after the "if" statement